src/build.js

```diff
@@ -0,0 +1,20 @@
+import QUnit from "core";
+import diff from "../external/jsdiff/jsdiff";
+import dump from "dump";
+import equiv from "equiv";
+import { begin, done, testStart, testDone, log } from "../reporter/html";
```

Comment:
This should disappear if we undo the conversion of the reporter to a ES6 module.